src/core/cuda_utils.h

```diff
@@ -47,7 +47,8 @@ namespace nvidia { namespace inferenceserver {
 
 #ifndef TRITON_ENABLE_GPU
 using cudaStream_t = void*;
-#endif  // TRITON_ENABLE_GPU
+using cudaEvent_t = void*;
```

Comment:
I don't think `cudaEvent_t` is mentioned in cuda_utils?

Reply:
You're right. Removed.